clients/web/test/spec/modelSpec.js

```diff
@@ -1,11 +1,20 @@
 /**
+ * Intercept window.location.assign calls so we can test the behavior of,
+ * e.g. download directives that occur from js.
+ */
+(function () {
+    window.location.assign = function (url) {
+        girderTest._redirect = url;
+    };
+}(window.location.assign));
```

Comment:
Why are we passing `window.location.assign` to a function that does not take any arguments?